commons/com.b2international.index/src/com/b2international/index/IndexClientFactory.java

```diff
@@ -158,6 +168,10 @@
 	 * The default cluster.name value for embedded nodes and tcp based clients.
 	 */
 	String DEFAULT_CLUSTER_NAME = "elastic-snowowl";
+	
+	int DEFAULT_BULK_ACTIONS_SIZE = 10_000;
+			
+	int DEFAULT_BULK_ACTIONS_SIZE_IN_MB = 10;
```

Comment:
I'm wondering if we should go a under the posted limit, in case the final request size goes over 10 MB with some overhead not accounted for. 🤔 

For future reference: https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/aes-limits.html#network-limits

Reply:
BulkProcessor should handle that well, I think.

Also, the smallest AWS ES instance limit is `10 MiB`, which is `10.048576 MB`.
If we limit ourselves to 10 MB by default, that should be less than the AWS 10 MiB limit.

Reply:
After a quick discussion, we have decided to set the limit to 9 MB by default to allow a bit of headroom for other parts in the HTTP request.